index.js

```diff
@@ -61,7 +61,7 @@ export function locatePathSync(
 
 	for (const path_ of paths) {
 		try {
-			const stat = statFunction(path.resolve(cwd, path_));
+			const stat = statFunction(path.resolve(cwd, path_), { throwIfNoEntry: false });
 
```

Comment:
Probably good to do a:

```js
if (!stat) {
  continue;
}
```

here to avoid calling the function if stat failed.